src/objects.js

```diff
@@ -1,41 +1,65 @@
 const createPerson = (name, age) => {
   // your code here
+  return {
+    name: name,
+    age: age,
+  };
 };
 
 const getName = (object) => {
   // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
   // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
   // your code here
+  return (object.hasOwnProperty(property));
 };
 
+
 const isOver65 = (person) => {
   // your code here
+  if (person.age > 65) {
+    return true;
+  }
+  return false;
 };
 
 const getAges = (people) => {
   // your code here
+  return people.map((person) => {
+    return person.age;
+  });
 };
 
 const findByName = (name, people) => {
   // your code here
+  return people.find((person) => name === person.name);
 };
 
 const findHondas = (cars) => {
   // your code here
+  return cars.filter(car => car.manufacturer === 'Honda');
 };
 
+
 const averageAge = (people) => {
   // your code here
+  return people.reduce(avAg => people.age / people.length);
 };
 
 const createTalkingPerson = (name, age) => {
```

Comment:
This test is the second of your failing tests - good attempt at using template strings however the syntax is slightly wrong, you should use backticks` instead of quotes. More can be found in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals